tests/rptest/tests/shadow_indexing_admin_api_test.py

```diff
@@ -0,0 +1,142 @@
+# Copyright 2021 Redpanda Data, Inc.
+#
+# Licensed as a Redpanda Enterprise file under the Redpanda Community
+# License (the "License"); you may not use this file except in compliance with
+# the License. You may obtain a copy of the License at
+#
+# https://github.com/redpanda-data/redpanda/blob/master/licenses/rcl.md
+
+from rptest.clients.kafka_cat import KafkaCat
+from rptest.services.cluster import cluster
+from rptest.tests.redpanda_test import RedpandaTest
+from rptest.services.redpanda import RedpandaService, SISettings
+
+from rptest.services.admin import Admin
+from rptest.archival.s3_client import S3Client
+from rptest.clients.types import TopicSpec
+from rptest.clients.rpk import RpkTool
+from rptest.clients.kafka_cli_tools import KafkaCliTools
+from rptest.util import (
+    segments_count,
+    produce_until_segments,
+    wait_for_segments_removal,
+    firewall_blocked,
+)
+
+import xxhash
+from collections import namedtuple, defaultdict
+import time
+import os
+import json
+import traceback
+import uuid
+import sys
+import re
+
+# Log errors expected when connectivity between redpanda and the S3
+# backend is disrupted
+CONNECTION_ERROR_LOGS = [
```

Comment:
we have the same list of errors in `archival_test.py` and `shadow_indexing_firewall_test.py` already. Maybe let's move it to `redpanda.py`?

Reply:
it's better be added separately, since I'll have to change all archival and SI tests and this will be out of scope